posts/2019-07-18-microprofile-30-developer-experience.adoc

```diff
@@ -10,8 +10,9 @@ blog_description: "MicroProfile 3.0 provides improvements to the developer exper
 ---
 = MicroProfile 3.0 with improved developer experience on Open Liberty 19.0.0.7
 Laura Cowen <https://github.com/lauracowen>
-
-Welcome to MicroProfile 3.0 on Open Liberty! MicroProfile 3.0 provides improvements to the developer experience.  MicroProfile Rest Client provides simplified configuration of SSL authentication and multiple client interfaces, MicroProfile Health Check separates liveness and readiness endpoints so that Kubernetes can check for both, and MicroProfile Metrics provides multi-dimensional metrics and simpler counters. In addition, Open Liberty now supports the PostgreSQL relational database. Give it a try in link:/about/[Open Liberty] 19.0.0.7.
+:imagesdir: /
+:url-prefix:
+:url-about: /Welcome to MicroProfile 3.0 on Open Liberty! MicroProfile 3.0 provides improvements to the developer experience.  MicroProfile Rest Client provides simplified configuration of SSL authentication and multiple client interfaces, MicroProfile Health Check separates liveness and readiness endpoints so that Kubernetes can check for both, and MicroProfile Metrics provides multi-dimensional metrics and simpler counters. In addition, Open Liberty now supports the PostgreSQL relational database. Give it a try in link:/about/[Open Liberty] 19.0.0.7.
```

Comment:
Does this need to be separated into two lines so `:url-about:` is blank like the other posts?

Reply:
Yep, thanks for catching this.

Reply:
Fixed and checked all the other posts for the same issue.